packages/fast-components/src/badge/badge.ts

```diff
@@ -0,0 +1,52 @@
+import { attr, FastElement } from "@microsoft/fast-element";
+
+export enum BadgeSize {
+    small = "small",
+    medium = "medium",
+    large = "large",
+}
+
+let lastBadgeSize: string;
+
+export class Badge extends FastElement {
+    @attr({ attribute: "fill" })
+    public fill: string;
+
+    // Revisit this once we have a better
+    // story for ensuring proper contrast
+    // from author defined `fill`
+    @attr({ attribute: "color" })
+    public color: string;
+
+    @attr
+    public size: BadgeSize;
+    private sizeChanged(): void {
```

Comment:
@EisenbergEffect looking at observable it seems like it would be trivial to invoke this change handler with a previous value (and "current" value, if we wanted) - that would help in a situation like this where we need to know what the value *was* - do you have concerns with that? If not I think it would be a useful feature.

Reply:
@nicholasrice is there something actionable here?

Reply:
not in the interim :) Was just hijacking your code to make Rob aware of a potential API improvement.